app/views/excursions/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <ul>
   <% @excursions.each do |excursion| %>
     <li>
-      <strong> <%= excursion.name %> </strong> |<%= excursion.description %>
+      <strong> <%= link_to excursion.name, excursion_path(excursion.id) %> </strong> |<%= excursion.description %>
```

Comment:
"excursion_path(excursion) "   delete the id, ruby knows that is the id (it was one of the questions on the lecture, i think)